tests/pages/perconaPlatformPage/perconaPlatformPage.js

```diff
@@ -74,10 +75,13 @@ module.exports = {
     I.seeTextEquals('', this.elements.emailValidation);
   },
 
-  connectToPortal(token, serverName = 'Test Server') {
+  connectToPortal(token, serverName = 'Test Server', connectedViaPortalApi = false) {
```

Comment:
connectedViaPortalApi, change please to something that describes it better, probably isPublicAddressSet